packages/core/src/quote/Quote.ts

```diff
@@ -103,7 +104,11 @@ export async function verifyAttesterSignedQuote(
 ): Promise<void> {
   const { attesterSignature, ...basicQuote } = quote
   await verifyDidSignature({
-    signature: attesterSignature,
+    signature: Crypto.coToUInt8(attesterSignature.signature),
+    keyUri:
+      attesterSignature.keyUri ??
+      // accept the old did signature format where keyUri was keyId
+      (attesterSignature as unknown as { keyId: DidResourceUri }).keyId,
```

Comment:
I reimplemented the backwards compatibility for the old DidSignature interface in its consumers. We could also drop that though, it's been a while and we have tons of breaking changes anyway?

Reply:
I think last time we had to put in specifically in the SDK because this function was called automatically before the application had any chance to update the data, like during decryption of a message. I am not sure this is still the case, I think Timo decoupled the decryption from verification. Still this could be a problem for long-living data objects, like on DIDsign. It could be much cheaper to just keep this in.

Reply:
moving this to the deserialiser is a good compromise I think